src/components/selection_input/selection_input.ts

```diff
@@ -83,6 +85,10 @@ export class SelectionInput extends Component<Props, SpreadsheetChildEnv> {
     onSelectionRemoved: { type: Function, optional: true },
     colors: { type: Array, optional: true, default: [] },
   };
+  static defaultProps = {
+    readonly: false,
```

Comment:
nitpick: it's a bit useless , undefined is already falsy. And if you have a default props, the props should be typed as non-optional in `Props` (because it will always have a value in practice)